x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/vectors/TextEmbeddingWeightedTokenBuilderTests.java

```diff
@@ -26,7 +26,7 @@
 import static org.hamcrest.Matchers.hasSize;
 import static org.hamcrest.core.IsInstanceOf.instanceOf;
 
-public class TextEmbeddingQueryVectorBuilderTests extends AbstractQueryVectorBuilderTestCase<TextEmbeddingQueryVectorBuilder> {
+public class TextEmbeddingWeightedTokenBuilderTests extends AbstractQueryVectorBuilderTestCase<TextEmbeddingQueryVectorBuilder> {
```

Comment:
Not a big deal, but what's the motivation for this rename?

Reply:
IntelliJ may have been overly aggressive with Refactor/Rename